### PR TITLE
DP-3665 Add example GenerateRecordSerializersExample

### PR DESCRIPTION
**Pull request by akmalxpert**

### CODE REVIEW

File: src/main/java/com/northconcepts/datapipeline/examples/cookbook/GenerateRecordSerializersExample.java [hunk 1]
```diff
@@ -0,0 +1,15 @@
+package com.northconcepts.datapipeline.examples.cookbook;
+
+import com.northconcepts.datapipeline.foundations.tools.GenerateRecordSerializers;
+
+import java.util.HashSet;
+
+public class GenerateRecordSerializersExample {
+    public static void main(String[] args) throws Throwable {
+        GenerateRecordSerializers serializers = new GenerateRecordSerializers();
```
Comment (DeleTaylor) [inline src/main/java/com/northconcepts/datapipeline/examples/cookbook/GenerateRecordSerializersExample.java]: Let's move this declaration down to just before it's needed/used.

File: src/main/java/com/northconcepts/datapipeline/examples/cookbook/GenerateRecordSerializersExample.java [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+        HashSet<String> types = new HashSet<>();
+        types.add("com.northconcepts.datapipeline.foundations.datamapping.DataMapping");
+        types.add("com.northconcepts.datapipeline.foundations.pipeline.input.CsvPipelineInput");
```
Comment (DeleTaylor) [inline src/main/java/com/northconcepts/datapipeline/examples/cookbook/GenerateRecordSerializersExample.java]: Let's create a couple static inner classes in this class to replace these two.  Let's make them simple POJOs -- nothing too complicated that hides the main purpose of the example.

**Review by DeleTaylor — CHANGES_REQUESTED** [review]

File: src/main/java/com/northconcepts/datapipeline/examples/cookbook/GenerateRecordSerializersExample.java (hunk 1 above)
Comment (akmalxpert) [reply src/main/java/com/northconcepts/datapipeline/examples/cookbook/GenerateRecordSerializersExample.java]: done

File: src/main/java/com/northconcepts/datapipeline/examples/cookbook/GenerateRecordSerializersExample.java (hunk 2 above)
Comment (akmalxpert) [reply src/main/java/com/northconcepts/datapipeline/examples/cookbook/GenerateRecordSerializersExample.java]: done

**Review by DeleTaylor — APPROVED** [review]

**Pull request merged**